aspnetcore/fundamentals/host/web-host.md

```diff
@@ -106,7 +141,7 @@ When setting up a host, [Configure](/dotnet/api/microsoft.aspnetcore.hosting.web
 [WebHostBuilder](/dotnet/api/microsoft.aspnetcore.hosting.webhostbuilder) relies on the following approaches to set the host configuration values:
 
 * Host builder configuration, which includes environment variables with the format `ASPNETCORE_{configurationKey}`. For example, `ASPNETCORE_ENVIRONMENT`.
-* Explicit methods, such as [HostingAbstractionsWebHostBuilderExtensions.UseContentRoot](/dotnet/api/microsoft.aspnetcore.hosting.hostingabstractionswebhostbuilderextensions.usecontentroot).
+* [HostingAbstractionsWebHostBuilderExtensions](/dotnet/api/microsoft.aspnetcore.hosting.hostingabstractionswebhostbuilderextensions), such as [UseContentRoot](/dotnet/api/microsoft.aspnetcore.hosting.hostingabstractionswebhostbuilderextensions.usecontentroot) and [UseConfiguration](/dotnet/api/microsoft.aspnetcore.hosting.hostingabstractionswebhostbuilderextensions.useconfiguration).
```

Comment:
Remove the name of the extension method class, just call it "Extensions such as...".

Reply:
Yeah, this UseConfiguration mention needs some way to tie into the other discussion of UseConfiguration later in the doc.